src/map/lua/luautils.h

```diff
@@ -142,6 +142,15 @@ namespace luautils
     auto  GetNPCByID(uint32 npcid, sol::object const& instanceObj) -> std::optional<CLuaBaseEntity>;
     auto  GetMobByID(uint32 mobid, sol::object const& instanceObj) -> std::optional<CLuaBaseEntity>;
     auto  GetEntityByID(uint32 mobid, sol::object const& instanceObj) -> std::optional<CLuaBaseEntity>;
+    /**
+     * Returns the npcs with a name matching the given pattern.
+     *
+     * @param pattern The pattern used to match the npc name. We use % as wildcard for consistency
+     * with other methods that perform patter matching.
+     * E.g: %anto% matches Shantotto and Canto-anto
+     */
```

Comment:
This docstring can be moved to the cpp file, tools will still get it

Reply:
Moved